clientapp/src/components/TracerForm.jsx

```diff
@@ -116,12 +121,16 @@ export class TracerForm extends React.Component {
     }
 
     onStartDateChange(ev) {
-        const newStartDate = ev.target.value;
+        var startDate = document.getElementById("start-date").value;
+        var startTime = document.getElementById("start-time").value;
```

Comment:
can we remove the seconds from the time? 
![image](https://user-images.githubusercontent.com/16966368/88841018-330ad600-d192-11ea-9ed2-b594f16c3423.png)